docs/tfc_migration.md

```diff
@@ -0,0 +1,219 @@
+# State Migration
+
+This is a guide for migrating your existing terraform state into OTF.
+
+## Migrating from Terraform Cloud / Enterprise
+
+If you're currently using Terraform Cloud or Terraform Enterprise, you are
+either using the [`remote` backend](https://developer.hashicorp.com/terraform/language/settings/backends/remote) or
+the newer [`cloud` block](https://developer.hashicorp.com/terraform/cli/cloud/settings). See the relevant instructions below.
+
+### Cloud block migration
+
+1. If you're using the the newer `cloud` block, your existing configuration will look something like this:
+
+		terraform {
+		  cloud {
+			hostname = "app.terraform.io"
+			organization = "automatize"
+
+			workspaces {
+			  name = "my-workspace"
+			}
+		  }
+		}
+
+1. Temporarily update the configuration to use `remote` instead:
+
+		terraform {
+		  backend "remote" {
+			hostname = "app.terraform.io"
+			organization = "automatize"
+
+			workspaces {
+			  name = "my-workspace"
+			}
+		  }
+		}
+
+	!!! note
+		This step is necessary because `terraform` does not allow state to be
+		migrated when using the cloud block configuration. Once you've migrated
+		the state you can re-introduce the cloud block (see below).
+
+1. Remove the `.terraform` directory:
+
+		rm -r .terraform
+
+1. Then follow the [Remote backend migration](#remote-backend-migration) instructions below.
+
+### Remote backend migration
+
+1. If you're using the `remote` backend, your existing configuration will look something like this:
+
+		terraform {
+		  backend "remote" {
+			hostname = "app.terraform.io"
+			organization = "automatize"
+
+			workspaces {
+			  name = "my-workspace"
+			}
+		  }
+		}
+
+1. To migrate to OTF you only need to update the hostname:
```

Comment:
Did that really work for you? 😮 
I tried this and always got rejected with "migrating form 'cloud' to other backends is not supported right now'. 

That's why I went with the cumbersome API workaround I shared some time ago. Did something change on the terraform side?

I mean great if it works that easily now, just wondering 😅

Reply:
That error only occurs when using the `cloud` block, not the `remote` backend. Hence the more long-winded instructions for migrating from the `cloud` block.